site/gatsby-site/src/scripts/import-classifications-csv.ts

```diff
@@ -0,0 +1,724 @@
+
+    // Validate the structure
+    if (!mapping.fieldMappings) {
+      throw new Error(`Missing "fieldMappings" section in the mapping file: ${filePath}`);
+    }
+
+    if (!mapping.valueMappings) {
+      throw new Error(`Missing "valueMappings" section in the mapping file: ${filePath}`);
+    }
+
+    // Warn if sections are empty
+    if (Object.keys(mapping.fieldMappings).length === 0) {
+      logger.warning(`"fieldMappings" section is empty in the mapping file: ${filePath}`);
+    }
+
+    if (Object.keys(mapping.valueMappings).length === 0) {
+      logger.warning(`"valueMappings" section is empty in the mapping file: ${filePath}`);
+    }
+
+    logger.info(`Loaded ${Object.keys(mapping.fieldMappings).length} field mappings and ${Object.keys(mapping.valueMappings).length} value mapping categories`);
+    return mapping;
+  }
+  catch (error) {
+    logger.error(`Error reading mapping file: ${(error as Error).message}`);
+    throw error;
+  }
+}
+
+/**
+ * Fetch taxonomy definition from GraphQL API
+ */
+async function fetchTaxonomyDefinition(
+  endpoint: string,
+  namespace: string,
+  sessionToken: string
+): Promise<TaxonomyField[]> {
+  const spinner = ora(`Fetching taxonomy definition for namespace: ${namespace}`).start();
+  const client = createApolloClient(endpoint, sessionToken);
+
+  const query = gql`
+    query GetTaxonomyDefinition($filter: TaxaFilterType!) {
+      taxa(filter: $filter) {
+        field_list {
+          mongo_type
+          short_name
+          permitted_values
+        }
+      }
+    }
+  `;
+
+  const variables = {
+    filter: {
+      namespace: {
+        EQ: namespace
+      }
+    }
+  };
+
+  try {
+    const result = await client.query({
+      query,
+      variables,
+      fetchPolicy: 'no-cache'
+    });
+
+    // Check for GraphQL errors
+    if (result.errors && result.errors.length > 0) {
+      const errorMessages = result.errors.map(e => e.message).join(', ');
+      spinner.fail(`GraphQL errors when fetching taxonomy: ${errorMessages}`);
+      throw new Error(`GraphQL errors: ${errorMessages}`);
+    }
+
+    if (!result.data?.taxa?.field_list || result.data.taxa.field_list.length === 0) {
+      spinner.fail(`No taxonomy definition found for namespace: ${namespace}`);
+      throw new Error(`No taxonomy definition found for namespace: ${namespace}`);
+    }
+
+    spinner.succeed(`Loaded ${result.data.taxa.field_list.length} taxonomy fields`);
+    return result.data.taxa.field_list;
+  }
+  catch (error: any) {
+    spinner.fail(`Error fetching taxonomy definition: ${error.message}`);
+    throw error;
+  }
+}
+
+/**
+ * Convert value based on MongoDB type
+ */
+function convertValueByMongoType(value: string, mongoType: string): any {
+  switch (mongoType) {
+    case 'string':
+      return value.trim();
+    default:
+      throw new Error(`Unsupported mongo_type: ${mongoType}. Supported types are: string, number, boolean, date`);
```

Comment:
Why does this say that string, number, and boolean are supported when the only case handled here is string?

Reply:
Yeah I starteed with more data types but realized we only need `string` for now(); Fixed.